src/services/carSaleTracking/bulkDealerFeedback.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiCallOptions, postData } from "../../base"
+
+export const postBulkDealerFeedback = async ({
+  dealerId,
+  elements,
+  options = {},
+}: {
+  dealerId: number
+  elements: {
+    listingId: number
+    purchaseConfirmed: boolean
+  }[]
+  options?: ApiCallOptions
+}): Promise<void> => {
+  return postData({
+    path: `dealers/${dealerId}/listings/car-sale/dealer-feedback`,
+    body: {
+      elements,
+    },
+    options,
```

Comment:
I think in options you need the authorized Request.

Reply:
IIRC it's only one of the purposes of the options they are also used to set custom fetch options.
Regarding auth I would assume that the request needs authentication as the single feedback one does @Marine-Berthier can you check that with backend and add it if needed

Reply:
Yes, will do !